pysite/base_route.py

```diff
@@ -24,7 +27,10 @@ class RouteView(BaseView):
     path = None  # type: str
 
     @classmethod
-    def setup(cls: "RouteView", blueprint: Blueprint):
+    def setup(cls: "RouteView", manager: "pysite.route_manager.RouteManager", blueprint: Blueprint):
```

Comment:
this method especially could use a docstring to explain  what's going on with the whole chain of inheritance thing.